trick_source/java/test/src/trick/common/utils/LogHeaderReaderTest.java

```diff
@@ -33,7 +33,7 @@ public void tearDown() throws Exception {
 
 	@Test(expected = FileNotFoundException.class)
 	public void testGetContentsWithNotFoundFile() throws FileNotFoundException, IOException {
-		DataReader reader = new LogHeaderReader("test" + System.getProperty("file.separator") + "resources" + System.getProperty("file.separator") + "unknowFile.header");
+		DataReader reader = new LogHeaderReader("resources" + System.getProperty("file.separator") + "unknowFile.header");
```

Comment:
While you're in here, I would replace all occurrences of `System.getProperty("file.separator")` with `File.separator` (from `java.io.File`). See [this thread](https://stackoverflow.com/questions/8075373/file-separator-vs-filesystem-getseparator-vs-system-getpropertyfile-separato), particularly the second answer.